packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -23,6 +23,7 @@ import styles from './style.scss';
 import BlockListBlock from './block';
 import BlockListAppender from '../block-list-appender';
 import BlockInsertionPoint from './insertion-point';
+import { __ } from '@wordpress/i18n';
```

Comment:
This should go into the `WordPress dependencies` section.

Reply:
Thanks! Didn't notice that. Fixed in f0f6ce3.